### ISSUE TITLE
how to run with gpt3.5

### ISSUE THREAD

**Issue by m0zzaR** [smol-ai/developer#25]
i have access to gpt4 but i'd rather not use it as I could imagine smol-developer, especially in its early development stage, would use a lot of token's and make many requests to ChatGPT.

I am trying to run this command:
~~~
modal run main.py --prompt prompt.md --model=gpt-3.5
~~~

I am also running into the issue where even if I do run it normally with:
~~~
modal run main.py --prompt prompt.md
~~~
It gives me this error:
~~~
InvalidRequestError: The model: `gpt-4` does not exist
~~~

Yes, I have access to GPT4 aswell as paid rate usage and yes I have access to the modal library. 
Does anyone know of a place within the smol-developer repository where i can change the model to gpt3.5?

**Comment by m0zzaR** [comment]
Update: sorry, all i had to do was update the 'openai_model' variable to 'gpt-3.5-turbo', and it generated a program. I thought id just leave this issue here for anyone else that runs into the same problem. 

Nonetheless I still cannot use smol-developer with gpt4, any ideas why?

**Comment by beingforthebenefit** [comment]
I have the same issue. "`gpt-4` does not exist". I have a credit card on file for the API and have ChatGPT Plus with access to ChatGPT 4. I even recreated the API key after upgrading to ChatGPT Plus. Still, I'm reduced to using `gpt-3.5-turbo`.

**Comment by hjdesulme** [comment]
Someone correct me if I'm wrong, but I believe that having access to GPT-4 via ChatGPT Plus is not the same as having API access to the `gpt-4` model. In order to get API access, you must join the [GPT-4 API waitlist](https://openai.com/waitlist/gpt-4-api).

**Comment by beingforthebenefit** [comment]
@hjdesulme You are absolutely correct. Thank you for the link--I'm now on the waitlist

**Comment by swyxio** [comment]
yes - there's a waitlist. the official way to get off the waitlist is to contribute something to https://github.com/openai/evals/ 

i may actually open up a smol.ai api path just for testing, with some rate limits, so that y'all can test it out properly since GPT4 is so much better at code gen. TODO list for me
